### PR TITLE
Add theming, start on basic layout, create connect wallet modal

### PR DESCRIPTION
**Pull request by neilmtan**
Just pure UI stuff as well as some theming scaffolding to take advantage of MUI.

### CODE REVIEW

File: bs-web-app/package.json [hunk 1]
```diff
@@ -13,6 +13,8 @@
     "@types/node": "^12.0.0",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-prettier": "^4.0.0",
```
Comment (jierlich) [inline bs-web-app/package.json:17]: Shouldn't these two also be dev deps?

**Review by jierlich — APPROVED** [review]
Solid foundation. I'd appreciate if you split commits up a bit more in the future for easier reviewing, but understand that it's kind of annoying with these early set up PRs

**Comment by jierlich** [comment]
Also, for frontend changes it'd be great to have screenshots in the PR for easier reviewing. Especially once we start getting to more specific changes

File: bs-web-app/package.json (hunk 1 above)
Comment (neilmtan) [reply bs-web-app/package.json:17]: It acts up when you put it in dev for some reason. Will add it back if it becomes an issue.

**Pull request merged**